data/project.cpp

```diff
@@ -47,12 +47,30 @@ void Project::accept(Visitor &visitor) const
 
 void Project::add(std::unique_ptr<File> file)
 {
+    // TODO: It could be better if there would be just std::map<QString, std::unique_ptr<File>> and no vector,
```

Comment:
sounds good, especially that this model is never used to display Files in "adding order"